girder/cumulus/server/cluster.py

```diff
@@ -160,6 +163,22 @@ def _create_ec2(self, params, body):
 
         return cluster
 
+    def _create_ansible(self, params, body):
+
+        self.requireParams(['name', 'template', 'profile'], body)
+
```

Comment:
In this case we use template to pass in extra variables needed by ansible to run the playbook. Theoretically any variable can be passed in,  but practically we just pass in instance type, number of instances etc.